crypt.go

```diff
@@ -1030,3 +1031,9 @@ func NameLegacyLUKS2ContainerKey(devicePath string, keyslot int, newName string)
 
 	return nil
 }
+
+func TestLUKS2ContainerKey(devicePath string, key []byte) bool {
+	cmd := exec.Command("cryptsetup", "open", "--test-passphrase", "--key-file", "-", devicePath)
```

Comment:
Rather than use `exec.Command` here, it might be worthwhile exposing an API for this from the `internal/luks2` package which this would then just wrap around. That's generally what we're doing with other APIs that require a call to cryptsetup.